helm/nginx-ingress-controller-app/templates/controller-service-internal.yaml

```diff
@@ -3,7 +3,9 @@ apiVersion: v1
 kind: Service
 metadata:
   annotations:
+  {{- if .controller.service.externalDNS.enabled }}
```

Comment:
This will disable it by default everywhere, right? IMO we should have a default in `values.yaml` which sets this to true to not break current expectation.

Reply:
This should also fix the lint error.